src/amo/components/Home/index.js

```diff
@@ -26,10 +26,9 @@ import Icon from 'ui/components/Icon';
 import './styles.scss';
 
 export const FEATURED_COLLECTIONS = [
+  { slug: 'refined-reading', username: 'mozilla' },
```

Comment:
for some reason I don't see this collection showing up on the frontend...do you?

Reply:
oh I guess b/c a version of this collection is not on dev...

Reply:
Just wondering - do you have permissions to create this on dev?

Reply:
no 😥 can you or @kumar303 create it by chance?

Reply:
@wagnerand created the collection 🎉

Reply:
FTR: we (devs) can create a collection under our own account and ask an admin to transfer this collection to the `mozilla` account. We could also setup a VPN and access the admin page :)